src/connector/mysql/conversion.rs

```diff
@@ -89,7 +90,7 @@ impl<'a> From<ParameterizedValue<'a>> for MyValue {
         match pv {
             ParameterizedValue::Null => MyValue::NULL,
             ParameterizedValue::Integer(i) => MyValue::Int(i),
-            ParameterizedValue::Real(f) => MyValue::Float(f.to_f64().expect("Decimal is not a f64.")),
+            ParameterizedValue::Real(f) => MyValue::Double(f.to_f64().expect("Decimal is not a f64.")),
```

Comment:
What happens here if the underlying column is a float, we write a double and then read it back?

Reply:
The database will cast it, most likely. I can write a test for it.

Reply:
Test written.